olm-catalog/serverless-operator/1.6.0/serverless-operator.v1.6.0.clusterserviceversion.yaml

```diff
@@ -68,7 +68,6 @@ metadata:
       Provides a collection of API's based on Knative to support deploying and serving
       of serverless applications and functions.
     repository: https://github.com/openshift-knative/serverless-operator
-    operators.operatorframework.io/internal-objects: '["knativeservings.operator.knative.dev"]'
```

Comment:
We shouldn't change the 1.6.0 CSV. If we want to fix this for 1.6.x, we'd need to create a new 1.6.1 release I believe.

Reply:
right, thanks!